### PR TITLE
license support added

### PR DESCRIPTION
**Pull request by HrishikeshGeedMS**
Added support to copy license file, if it is available with model files

### CODE REVIEW

File: training/model_management/src/run_model_preprocess.py [hunk 1]
```diff
@@ -9,7 +9,9 @@
 from azureml.model.mgmt.config import ModelFlavor
 from azureml.model.mgmt.processors.preprocess import run_preprocess
 from pathlib import Path
+import shutil
 
+LICENSE_FILE = 'LICENSE'
```
Comment (novaturient95) [inline training/model_management/src/run_model_preprocess.py:14]: may be use a regex?

File: training/model_management/src/run_model_preprocess.py [hunk 2]
```diff
@@ -59,5 +61,12 @@ def _validate_hf_transformers_args(args):
         _validate_hf_transformers_args(preprocess_args)
 
     run_preprocess(mlflow_flavor, model_path, mlflow_model_output_dir, **preprocess_args)
+
+    ## Adding support for license file if exist in model path
+    if os.path.exists(Path(model_path,LICENSE_FILE)):
+        shutil.copy(Path(model_path,LICENSE_FILE), mlflow_model_output_dir)
```
Comment (novaturient95) [inline training/model_management/src/run_model_preprocess.py:67]: This is specific to hugging face conversion. Please add changes in  hftransformers/convert.py

**Review by novaturient95 — CHANGES_REQUESTED** [review]

File: training/model_management/src/run_model_preprocess.py (hunk 1 above)
Comment (HrishikeshGeedMS) [reply training/model_management/src/run_model_preprocess.py]: Done

File: training/model_management/src/run_model_preprocess.py (hunk 2 above)
Comment (HrishikeshGeedMS) [reply training/model_management/src/run_model_preprocess.py]: Done

**Comment by novaturient95** [comment]
can we abandon this in favor of: https://github.com/Azure/azureml-assets/pull/463 ?

**Comment by skasturi** [comment]
Abandoning in favor of #463

**Pull request closed without merging**